PasswordVerifier.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace PasswordVerifier
+{
+    internal class Program
```

Comment:
class with name 'Program' does not give a proper understanding of what it does. Use a good naming convention.

Reply:
Done! Resolved on the new commit. Thank you!